Damnit, test failures:

```
2020-06-17T09:19:12.5708095Z [ERROR] Failures: 
2020-06-17T09:19:12.5725946Z [ERROR]   JaxRSTestCase.testSSE:272 1 expectation failed.
2020-06-17T09:19:12.5726587Z Expected content-type "text/event-stream" doesn't match actual content-type "".
2020-06-17T09:19:12.5726674Z 
2020-06-17T09:19:12.5727946Z [ERROR]   JaxRsConstructorPropertiesTestCase.testWrappedInServerSentEventMessage:38 1 expectation failed.
2020-06-17T09:19:12.5729351Z Response body doesn't match expectation.
2020-06-17T09:19:12.5733111Z Expected: a string containing "ssevalue"
2020-06-17T09:19:12.5735532Z   Actual: 
2020-06-17T09:19:12.5735651Z 
2020-06-17T09:19:12.5735724Z [INFO] 
```

--- a/integration-tests/main/src/main/java/io/quarkus/it/rest/ServerSentEventResource.java
+++ b/integration-tests/main/src/main/java/io/quarkus/it/rest/ServerSentEventResource.java
@@ -62,7 +62,8 @@ public class ServerSentEventResource {
 
     @GET
     @Path("/stream-xml")
-    @Produces("text/event-stream;element-type=text/xml")
+    @SseElementType(MediaType.TEXT_XML)
+    @Produces(MediaType.SERVER_SENT_EVENTS)
     public void sendXmlData(@Context SseEventSink sink) {
         // send a stream of few events
         try {

--- a/integration-tests/main/src/main/java/io/quarkus/it/rest/TestResourceForConstructorProperties.java
+++ b/integration-tests/main/src/main/java/io/quarkus/it/rest/TestResourceForConstructorProperties.java
@@ -13,6 +13,8 @@ import javax.ws.rs.sse.OutboundSseEvent;
 import javax.ws.rs.sse.Sse;
 import javax.ws.rs.sse.SseEventSink;
 
+import org.jboss.resteasy.annotations.SseElementType;
+
 @Path("/constructorproperties")
 public class TestResourceForConstructorProperties {
 
@@ -44,7 +46,8 @@ public class TestResourceForConstructorProperties {
 
     @GET
     @Path("/sse")
-    @Produces(MediaType.SERVER_SENT_EVENTS + ";element-type=" + MediaType.APPLICATION_JSON)
+    @SseElementType(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.SERVER_SENT_EVENTS)
     public void serverSentEvents(@Context SseEventSink sink) {
         VanillaJavaImmutableData data = new VanillaJavaImmutableData("sse", "ssevalue");
         try {

